### PR TITLE
fix examples of classes in stylesheet reading

### LINKED ISSUE #1433: [Bug]: Can't get classes to work
### What happened?

Project is generated with `phx.new`, `lvn.setup.config`, `lvn.setup.gen`.

`config/config.exs`:
```elixir
config :live_view_native_stylesheet,
  content: [
    swiftui: [
      "lib/**/swiftui/*",
      "lib/**/*swiftui*"
    ]
  ],
  output: "priv/static/assets"
```

`lib/my_app_web/live/home_live.swiftui.ex`:
```elixir
defmodule MyAppWeb.Styles.App.SwiftUI do
  use LiveViewNative.Stylesheet, :swiftui

  ~SHEET"""
    "red-title" do
      foregroundColor(.red);
      font(.title);
    end
  """
end
```

`lib/my_app_web/router.ex`:
```elixir
  scope "/", MyAppWeb do
    pipe_through :browser

    live "/", HomeLive
  end
```

`lib/my_app_web/components/layouts_swiftui/root.swiftui.neex`:
```elixir
<.csrf_token />
<Style url={~p"/assets/app.swiftui.styles"} />
<NavigationStack>
  <%= @inner_content %>
</NavigationStack>
```

`lib/my_app_web/live/home_live.swiftui.ex`:
```elixir
defmodule MyAppWeb.HomeLive.SwiftUI do
  use MyAppNative, [:render_component, format: :swiftui]

  def render(assigns, _) do
    ~LVN"""
    <Text class="red-title">Red Title Text</Text>
    """
  end
end
```
`http://localhost:4000/assets/app.swiftui.styles`:
```elixir
%{
  "alert(attr(\"title\"), isPresented: attr(\"presented\"), actions: :actions, message: :message)" => [
    {:alert, [],
     [
       {:__attr__, [], "title"},
       [
         isPresented: {:__attr__, [], "presented"},
         actions: :actions,
         message: :message
       ]
     ]}
  ],
  "bold(true)" => [{:bold, [], [true]}],
  "buttonStyle(.borderedProminent)" => [
    {:buttonStyle, [], [{:., [], [nil, :borderedProminent]}]}
  ],
  "controlSize(.large)" => [{:controlSize, [], [{:., [], [nil, :large]}]}],
  "font(.caption)" => [{:font, [], [{:., [], [nil, :caption]}]}],
  "foregroundStyle(.red)" => [{:foregroundStyle, [], [{:., [], [nil, :red]}]}],
  "frame(maxWidth: .infinity)" => [
    {:frame, [], [[maxWidth: {:., [], [nil, :infinity]}]]}
  ],
  "hidden()" => [{:hidden, [], []}],
  "listRowBackground(:empty)" => [{:listRowBackground, [], [:empty]}],
  "listRowInsets(EdgeInsets())" => [
    {:listRowInsets, [], [{:EdgeInsets, [], []}]}
  ],
  "navigationSubtitle(:subtitle)" => [{:navigationSubtitle, [], [:subtitle]}],
  "navigationTitle(:title)" => [{:navigationTitle, [], [:title]}],
  "sheet(isPresented: attr(\"presented\"), content: :content)" => [
    {:sheet, [],
     [[isPresented: {:__attr__, [], "presented"}, content: :content]]}
  ],
  "toolbar(content: :toolbar)" => [{:toolbar, [], [[content: :toolbar]]}]
}
```

Screenshot 2024-09-03 at 16 09 20

I can't find `red-title` styles from `http://localhost:4000/assets/app.swiftui.styles`. I'm not sure how to debug this further.

### Library Version

0.3.0

### Xcode Version

15.4

### Swift Version

5.10

### On which device or simulator are you running into the problem?

iPhone

### Target Device Operating System Version

17.5

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by BrooklinJazz**
closes #1433 

Semi-colons are not supported, and likely won't be until v.0.4.0. See: https://github.com/liveview-native/liveview-client-swiftui/issues/1421 for context.

This PR removes semi-colons from ~SHEET stylesheet examples.

I also improved the spacing of an inline style and fixed the custom color example while I was at it.

### CODE REVIEW

File: livebooks/markdown/stylesheets.md [hunk 1]
```diff
@@ -350,8 +350,8 @@ We group modifiers together within a class that can be applied to an element. He
 ```elixir
 ~SHEET"""
   "red-title" do
-    foregroundColor(.red);
-    font(.title);
+    foregroundColor(.red)
```
Comment (cmnstmntmn) [inline livebooks/markdown/stylesheets.md:353]: this be updated to `foregroundStyle`, right?

Comment (bcardarella) [reply livebooks/markdown/stylesheets.md:353]: Yup

Comment (BrooklinJazz) [reply livebooks/markdown/stylesheets.md:353]: Done

Comment (cmnstmntmn) [reply livebooks/markdown/stylesheets.md:353]: there are two other places, were it remained unchanged.

File: livebooks/markdown/stylesheets.md [hunk 2]
```diff
@@ -367,8 +367,8 @@ defmodule ServerWeb.Styles.App.SwiftUI do
 
   ~SHEET"""
     "red-title" do
-      foregroundColor(.red);
-      font(.title);
+      foregroundColor(.red)
```
Comment (cmnstmntmn) [inline livebooks/markdown/stylesheets.md:370]: to be switched to `foregroundStyle`

**Review by cmnstmntmn — APPROVED** [review]

**Pull request merged**